index.html

```diff
@@ -0,0 +1,42 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="./styles.css">
+    <title>My Portfolio</title>
+</head>
+<body>
+    <div>
+        <header>
+            <nav>
+                <ul>
+                    <p>John Doe</p>
+                    <img src="./img/hearder img/Union.png" alt="toggle-icon">
+                </ul>
+            </nav>
+        </header>
+        <section class="sec-container">
+            <div class="hero">
+                <h1>Hey There. I’m Alex</h1>
+                <h4>I’am a Software Developer</h4>
+                <p class="para">
+                   I can help you build a product , feature or website Look through some of my work 
+                   and experience! If you like what you see and have a project you need coded, 
+                   don’t hestiate to contact me.
+               </p>
```

Comment:
- [ ] If you look closely, you will see that this text is overlapping the background illustration. Kindly adjust the width of this text box such that it does not overlap.  

![image](https://user-images.githubusercontent.com/84156714/219630068-1a7d9135-dff4-4c65-ad4d-5a8ed11cde46.png)